transportable-udfs-examples/transportable-udfs-example-udfs/src/test/java/com/linkedin/transport/examples/TestBinaryDuplicateFunction.java

```diff
@@ -16,7 +16,15 @@
 import java.util.Map;
 import org.testng.annotations.Test;
 
-
+// Temporarily disable the tests for Trino. As the test infrastructure from Trino named QueryAssertions is used to
+// run these test for Trino, QueryAssertions mandatory execute the function with the query in two formats: one with
+// is the normal query (e.g. SELECT "binary_duplicate"(a0) FROM (VALUES ROW(from_base64('YmFy'))) t(a0);), the other
+// is with "where RAND()>0" clause (e.g. SELECT "binary_duplicate"(a0) FROM (VALUES ROW(from_base64('YmFy'))) t(a0) where RAND()>0;)
+// QueryAssertions verifies the output from both queries are equal otherwise the test fail.
+// However, the execution of the query with where clause triggers the code of VariableWidthBlockBuilder.writeByte() to create
+// the input byte array in Slice with an initial 32 byes capacity, while the execution of the query without where clause does not trigger
+// the code of VariableWidthBlockBuilder.writeByte() and create the input byte array in Slice with the actual capacity of the content.
+// Therefore, the outputs from both queries are different.
```

Comment:
+1 disabling the tests does not seem like the right course of action here

Reply:
Actually are you able to run the failed test with `QueryAssertion` now? It was not yet clear why `QueryAssertion` failed previously with the old way using `SqlScalarFunction`. Annotation-driven functions had no issue with `QueryAssertion`.

Reply:
The test still fails with `QueryAssertions`. Briefly `QueryAssertions` requires to compare the outputs from two queries with the same UDF, however in case of binary UDF, two queries results in the execution of different code paths in Trino which makes the inputs to UDF are different, so the outputs of the queries are different. The details are added in the comments of the code. 
I have created a TODO issue to reenable them after the root cause is found in Trino and fixed as shown in https://github.com/linkedin/transport/issues/131